src/atoms/Link/Link.native.js

```diff
@@ -0,0 +1,71 @@
+import React, { useContext, useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import Text from '../Text';
+import styled, { ThemeContext } from 'styled-components/native';
+
+const StyledTouchable = styled.TouchableOpacity`
+  align-self: flex-start;
+`;
+
+const resolveSizeToTextProps = (linkSize) => {
+  //TODO: Add lineHeight prop for all cases
+  switch (linkSize) {
+    case 's':
+      return { size: 'xs', weight: 'regular' };
```

Comment:
For `case 's'`, it should return `{ size: 's', weight: 'regular' }`?

Reply:
No. I had to declare a mapping because in the designs there are three sizes each mapping to text size of xs,m and l. Nothing for s